### ISSUE TITLE
Doesn't work with Giraffe 6, .NET 6

### ISSUE THREAD

**Issue by olivercoad** [Nhowka/Elmish.Bridge#39]
Trying to upgrade to Giraffe 6. Getting an error because it doesn't include Ply.

```
server: info: Microsoft.AspNetCore.Hosting.Diagnostics[1]
server:       Request starting HTTP/1.1 GET http://localhost:8080/socket/clientbridge - -
server: fail: Giraffe.Middleware.GiraffeErrorHandlerMiddleware[0]
server:       An unhandled exception has occurred while executing the request.
server:       System.IO.FileNotFoundException: Could not load file or assembly 'Ply, Version=0.3.1.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
server:       
server:       File name: 'Ply, Version=0.3.1.0, Culture=neutral, PublicKeyToken=null'
server:          at Elmish.Bridge.Giraffe.server@17-1.Invoke(FSharpFunc`2 next, HttpContext ctx)
server:          at Giraffe.Core.chooseHttpFunc@120.MoveNext()
server:          at Giraffe.Core.chooseHttpFunc@120.MoveNext()
server:          at Giraffe.Core.chooseHttpFunc@120.MoveNext()
server:          at Giraffe.Middleware.Invoke@30.MoveNext()
server:          at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
server:          at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
server:          at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionMiddleware.InvokeCore(HttpContext context)
server:          at Giraffe.Middleware.Invoke@63-1.MoveNext()
server: info: Microsoft.AspNetCore.Hosting.Diagnostics[2]
server:       Request finished HTTP/1.1 GET http://localhost:8080/socket/clientbridge - - - 500 140 application/json;+charset=utf-8 1.3749ms
```

Workaround is to explicitly add `nuget Ply` as a dependency.

**Comment by Nhowka** [comment]
Published versions 6.0.0 of the libraries using net6.0. They should be available soon.

**Comment by olivercoad** [comment]
Thanks! Just tested version 6 without Ply and it works great.
